registry/tracecat_registry/templates/checkpoint/get_cp_xdr_incidents.yml

```
@@ -0,0 +1,23 @@
+type: action
+definition:
+  name: get_cp_xdr_incidents
```

```suggestion
  name: get_xdr_incidents
```

Our convention is not to need the tool name in the action name. Since in the library it already shows up as `integrations.checkpoint.some_action_name`